src/components/Layout/Partials/SocialLinks.tsx

```diff
@@ -1,19 +1,21 @@
 import * as React from "react";
 
+import {Config} from "../../../Config";
+
 import {GitIcon} from "../../Icons/GitIcon";
 import {BehanceIcon} from "../../Icons/BehanceIcon";
 import {LinkedinIcon} from "../../Icons/LinkedinIcon";
 
-export const SocialLinks = () => {
+export const SocialLinks: React.SFC<undefined> = () => {
     return (
         <div className="social-list">
-            <a href="#" className="social-list__item">
-               <GitIcon/>
+            <a href={Config.links.github} className="social-list__item" target="_blank">
```

Comment:
`rel="nofollow noopener"` for external links.
See https://habrahabr.ru/post/282880/